src/components/input/SelectNext.tsx

```diff
@@ -194,7 +194,7 @@ function SelectMain<T>({
         id={buttonId ?? defaultButtonId}
         variant="custom"
         classes={classnames(
-          'w-full flex',
+          'w-full flex justify-between',
```

Comment:
What is the `justify-between` for here? To replace the `grow` spacer below?

Reply:
Yes. The `text-overflow: ellipsis` didn't work on a `grow` element.